src/app/design/guidelines/content-containers/index.html

```diff
@@ -155,7 +155,28 @@ <h3 class="sky-subsection-heading">
   </h3>
 
   <p>
-    Popovers show small chunks of information that are only occasionally relevant to the user. If the content is relevant most of the time, it should be presented in-page instead. If the amount of content would require scrolling on a mobile device, use a flyout instead.
+    Popovers show small chunks of information that are only occasionally relevant to the user. Popovers are appropriate when:
   </p>
+	
+	<ul>
+		<li>The trigger is client-side, a direct result of a user interaction with an element (such as a click) that does not lead to a change in system state.</li>
+		<li>The content is directly related to the user's current task.</li>
+		<li>The content will not require scrolling on a mobile device.</li>
+	</ul>
+	
+	<p>If the content is relevant most of the time, present it <a routerLink="/design/guidelines/user-assistance" fragment="persistent-inline-help">in-page</a> instead. If the amount of content would require scrolling on a mobile device, use a <a routerLink="/components/flyout">flyout</a> instead.</p>
```

Comment:
As explained above, I'd change this to something like: "If the content needs to be visible most of the time, use persistent inline help instead. If the content requires scrolling on mobile devices, use a flyout instead."